mods/cnc/maps/nod05/nod05.lua

```diff
@@ -0,0 +1,319 @@
+
+YyyyTriggerFunction = function()
+	if not YyyyTriggerSwitch then
+		for type, count in pairs(Gdi2Units) do
+			MyActors = Utils.Take(count, GDI.GetActorsByType(type))
+			Utils.Do(MyActors, function(actor)
+				WaypointMovementAndHunt(actor, Gdi2Waypoints)
+			end)
+		end
+	end
+end
+
+ZzzzTriggerFunction = function()
+	if not ZzzzTriggerSwitch then
+		for type, count in pairs(Gdi1Units) do
+			MyActors = Utils.Take(count, GDI.GetActorsByType(type))
+			Utils.Do(MyActors, function(actor)
+				WaypointMovementAndHunt(actor, Gdi1Waypoints)
+			end)
+		end
+	end
+end
+
+Grd1TriggerFunction = function()
+	MyActors = Utils.Take(2, GDI.GetActorsByType('jeep'))
+	Utils.Do(MyActors, function(actor)
+		WaypointMovementAndHunt(actor, Gdi5Waypoints)
+	end)
+end
+
+Atk5TriggerFunction = function()
+	WaypointMovementAndHunt(GDI.GetActorsByType('mtnk')[1], Gdi12Waypoints)
+end
+
+Atk2TriggerFunction = function()
+	for type, count in pairs(Gdi1Units) do
+		MyActors = Utils.Take(count, GDI.GetActorsByType(type))
+		Utils.Do(MyActors, function(actor)
+			WaypointMovementAndHunt(actor, Gdi1Waypoints)
+		end)
+	end
+end
+
+Atk3TriggerFunction = function()
+	for type, count in pairs(Gdi2Units) do
+		MyActors = Utils.Take(count, GDI.GetActorsByType(type))
+		Utils.Do(MyActors, function(actor)
+			WaypointMovementAndHunt(actor, Gdi2Waypoints)
+		end)
+	end
+end
+
+Atk4TriggerFunction = function()
+	WaypointMovementAndHunt(GDI.GetActorsByType('jeep')[1], Gdi3Waypoints)
+end
+
+Atk6TriggerFunction = function()
+	WaypointMovementAndHunt(GDI.GetActorsByType('mtnk')[1], Gdi2Waypoints)
+end
+
+Atk1TriggerFunction = function()
+	Reinforcements.ReinforceWithTransport(GDI, 'tran', GDIReinforceUnits, { waypoint9.Location, waypoint26.Location }, nil,
+	function(transport, cargo)
+		transport.UnloadPassengers()
+		Utils.Do(cargo, function(actor)
+			IdleHunt(actor)
+		end)
+	end)
+end
+
+AutoTriggerFunction = function()
+	local units = AllUnits[DateTime.GameTime % #AllUnits + 1]
+	local waypoints = AllWaypoints[DateTime.GameTime % #AllWaypoints + 1]
+
+	for type, count in pairs(units) do
+		MyActors = Utils.Take(count, GDI.GetActorsByType(type))
+		Utils.Do(MyActors, function(actor)
+			WaypointMovementAndHunt(actor, waypoints)
+		end)
+	end
+end
+
+HuntTriggerFunction = function()
+	local list = GDI.GetGroundAttackers()
+	Utils.Do(list, function(unit)
+		IdleHunt(unit)
+	end)
+end
+
+WaypointMovementAndHunt = function(unit, waypoints)
+	Utils.Do(waypoints, function(waypoint)
+		unit.AttackMove(waypoint.Location)
+	end)
+	IdleHunt(unit)
+end
+
+InsertNodUnits = function()
+	Reinforcements.Reinforce(Nod, NodUnitsVehicle, { UnitsEntry.Location, UnitsRallyVehicle.Location }, 1)
+	Reinforcements.Reinforce(Nod, NodUnitsRocket, { UnitsEntry.Location, UnitsRallyRocket.Location }, 50)
+	Reinforcements.Reinforce(Nod, NodUnitsGunner, { UnitsEntry.Location, UnitsRallyGunner.Location }, 50)
+	Reinforcements.Reinforce(Nod, { 'mcv' }, { UnitsEntry.Location, UnitsRallyMCV.Location }, 100)
```

Comment:
Could the `mcv` move in with a bit delay? (So that it doesn't block your incoming army.)